assets/helpers/logger.js

```diff
@@ -27,3 +27,12 @@ export const logException = (ex: string, context?: Object): void => {
   }
 };
 
+export const logInfo = (message: string): void => {
+  Raven.captureMessage(
+    message,
+    {
+      level: 'info',
+    },
+  );
+};
+
```

Comment:
Could exports go at the bottom of the file?

Reply:
Fixed in d0da663